src/main/java/org/broadinstitute/hellbender/tools/spark/transforms/BaseRecalibratorSparkFn.java

```diff
@@ -45,7 +45,7 @@ public static RecalibrationReport apply( final JavaPairRDD<GATKRead, ReadContext
             return new ArrayList<>(Arrays.asList(bqsr.getRecalibrationTables()));
         });
 
-        final RecalibrationTables combinedTables = unmergedTables.reduce(RecalibrationTables::safeCombine);
+        final RecalibrationTables combinedTables = unmergedTables.reduce(RecalibrationTables::inPlaceCombine);
```

Comment:
Actually, I'm not certain if it's necessary.  Haven't gotten a straight answer from anyone yet.  Feel free to leave for now.